runtime/assembly/__tests__/mock-builder.ts

```diff
@@ -149,14 +166,29 @@ export namespace MockHelper {
         const digest = new Option<DigestItem[]>(MockHelper._getDigests());
         return new Header(hash69, blockNumber, hash255, hash255, digest);
     }
-
-    export function _getEmptyInherentInstance(): Inherent {
-        const timestamp: UInt64 = new UInt64(0);
-        const babeslot: UInt64 = new UInt64(0);
-        const finalnum: CompactInt = new CompactInt(0);
-        const headers: Header[] = [];
-
-        return new Inherent(timestamp, babeslot, finalnum, headers);
+    /**
+     * Returns an InherentData instance 
+     * Used internally in the mock builder
+     */
+    export function _getEmptyInherentInstance(): InherentData {
```

Comment:
I think that the name of this function should not longer be `empty`, but rather:
`_getInherentDataInstance`